chainbench/main.py

```diff
@@ -77,6 +93,20 @@ def cli(ctx: click.Context):
 @click.option("--target", default=None, help="Endpoint to test")
 @click.option("--run-id", default=None, help="ID of the test")
 @click.option("--notify", default=None, help="Notify when test is finished")
+@click.option(
+    "-dt",
```

Comment:
Not sure about this shortcut as I'm afraid it can further clash with something as `-dt` is too generic. Maybe let's leave only the long option for now?

Reply:
okay!